src/makeCalculator.js

```diff
@@ -37,7 +37,45 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, number) {
+      return callback(number);
+    },
+
+    add(b) {
+      calculator.result += b;
```

Comment:
The method inside the object so you need to use `this` instead of calculator
And do it the same to the following methods because `result` is internal property not in a different object